tgbot/parser/residue_parser.py

```diff
@@ -25,7 +26,8 @@ async def get_residue(article):
             response_data = response_json.get('data')
             response_product = response_data.get('products')[0]
             response_sizes = response_product.get('sizes')
-        except AttributeError and IndexError:
+        except AttributeError and IndexError as e:
+            logger.error(f'Произошла ошибка: {e}')
```

Comment:
Давай уберем эти логирования ошибок и напишем специальный хендлер . Нужно создать bot\error_handler.py там напишем обработчик 
```suggestion
import logging

from telegram import Update
from telegram.ext import ContextTypes


async def error_handler(update: Update, context: ContextTypes.DEFAULT_TYPE):
    """Log Error caused by Update.."""
    logging.error(msg="Exception while handling an update:", exc_info=context.error)
```

Далее этот обработчик  добавляем в бота application.add_error_handler(error_handler)